File: Tareas/Unidad2/a2253332351_practica04/37_DescuentoProducto.java
import java.util.Scanner;

public class DescuentoProducto {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        double precio;
        double porcentaje;
        double descuento;
        double totalPagar;

        System.out.print("Ingrese el precio del producto: ");
        precio = entrada.nextDouble();
        System.out.print("Ingrese el porcentaje de descuento: ");
        porcentaje = entrada.nextDouble();

        descuento = precio * (porcentaje / 100);
        totalPagar = precio - descuento;

        System.out.println("El descuento es: " + descuento);
        System.out.println("El total a pagar es: " + totalPagar);
    }
}

File: Tareas/Unidad2/a2253332351_practica04/37_DescuentoProducto.py
# DescuentoProducto.py

precio = float(input("Ingrese el precio del producto: "))
porcentaje = float(input("Ingrese el porcentaje de descuento: "))

descuento = precio * (porcentaje / 100)
totalPagar = precio - descuento

print(f"El descuento es: {descuento}")
print(f"El total a pagar es: {totalPagar}")
